addons/account/migrations/11.0.1.1/post-migration.py

```diff
@@ -91,21 +91,66 @@ def fill_account_invoice_line_total(env):
     rest_lines = line_obj.search([]) - empty_lines - simple_lines
     openupgrade.logger.debug("Compute the rest of the account.invoice.line"
                              "totals: %s" % len(rest_lines))
-    for line in rest_lines:
+
+    # We group invoice lines that have the same price_unit, discount,
+    # currency_id, quantity, product_id and invoice_line_tax_ids since they
+    # will have the same price_total
+    env.cr.execute("""
+        SELECT
+                STRING_AGG(id::CHARACTER varying, ',') id,
```

Comment:
Does uom not need consideration here? I don't know, it just seems it would

Reply:
Also, under what circumstances does the product matter, I get that it is used further down, but I don't think it does? I guess if there are a lot of lines for only a few products. Oh I answered my own uom question the same way. Clearly you don't need uom for the same reason you don't need product.

Reply:
I had a look and it seems to me product is only used for 'code' type computation.